src/components/HTMLEngineProvider/HTMLRenderers/AnchorRenderer.js

```diff
@@ -24,12 +26,11 @@ const AnchorRenderer = (props) => {
     const displayName = lodashGet(props.tnode, 'domNode.children[0].data', '');
     const parentStyle = lodashGet(props.tnode, 'parent.styles.nativeTextRet', {});
     const attrHref = htmlAttribs.href || '';
-    const internalNewExpensifyPath = (attrHref.startsWith(CONST.NEW_EXPENSIFY_URL) && attrHref.replace(CONST.NEW_EXPENSIFY_URL, ''))
-        || (attrHref.startsWith(CONST.STAGING_NEW_EXPENSIFY_URL) && attrHref.replace(CONST.STAGING_NEW_EXPENSIFY_URL, ''));
-    const internalExpensifyPath = attrHref.startsWith(CONFIG.EXPENSIFY.EXPENSIFY_URL)
-                                    && !attrHref.startsWith(CONFIG.EXPENSIFY.CONCIERGE_URL)
-                                    && attrHref.replace(CONFIG.EXPENSIFY.EXPENSIFY_URL, '');
-
+    const attrPath = lodashReplace(Url.getURLObject(attrHref).path, '/', '');
```

Comment:
Do we really need `lodashReplace` for this case? I don't think we are using `lodashReplace` till now so no need to increase the bundle size.

Instead, do. 
```suggestion
    const attrPath = lodashGet(Url.getURLObject(attrHref), 'path', '').replace('/', '');
```